packages/dd-trace/src/config.js

```diff
@@ -453,6 +453,7 @@ class Config {
       DD_DYNAMIC_INSTRUMENTATION_UPLOAD_INTERVAL_SECONDS,
       DD_ENV,
       DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED,
+      DD_EXPERIMENTAL_PROPAGATE_PROCESS_TAGS_ENABLED,
```

Comment:
Please add a test case for the env. As soon as that is the case, the CI should complain that the value is not in the supported-configurations.json file, so it would also have to be added there.

I guess I have to rework our linter to pick this up earlier :)